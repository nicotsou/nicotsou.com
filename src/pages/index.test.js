import React from 'react'
import Index from './index'
import { shallow } from 'enzyme'

it('should display a dummy page', () => {
  const wrapper = shallow(<Index />)
  expect(wrapper).toMatchSnapshot()
})
