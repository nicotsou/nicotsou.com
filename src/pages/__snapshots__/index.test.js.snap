// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display a dummy page 1`] = `
<Layout>
  <h1>
    Hi people
  </h1>
  <p>
    Welcome to your new Gatsby site.
  </p>
  <p>
    Now go build something great.
  </p>
  <Component
    to="/page-2/"
  >
    Go to page 2
  </Component>
</Layout>
`;
